### PR TITLE
Feature/kount

### PR DESCRIPTION
**Pull request by gabrielsimpsons**
* Kount inquiry and update implemented
* Use of change-case to format kount result attributes to camelCase
* Unit testing for new functions

### CODE REVIEW

**Review by wuon — APPROVED** [review]
LGTM!

File: test/unit.test.js [hunk 1]
```diff
@@ -320,6 +322,130 @@ describe('Unit Testing', () => {
...
-    })
-  })
+    });
+  });
+
+  describe('moneris.kountInquire', () => {
+    it('Should return an incomplete score', async () => {
+      const res = await moneris.kountInquire({
+        kount_merchant_id: '760000',
+        kount_api_key: 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiI3NjAwMDAiLCJhdWQiOiJLb3VudC4xIiwiaWF0IjoxNTU4MDQwODQ5LCJzY3AiOnsia2EiOm51bGwsImtjIjpmYWxzZSwiYXBpIjp0cnVlLCJyaXMiOnRydWV9fQ.y3_2yzd11-Y_F6_xzVsXI-NO1a7P6ldMjDnKzl5yBko', // 214 character max - This is a UNIQUE local identifier used by the merchant to identify the kount inquiry request
```
Comment (wuon) [inline test/unit.test.js:332]: is this supposed to be public?

File: index.js [hunk 2]
```diff
@@ -164,6 +188,17 @@ const send = async (data, type, configuration) => {
     out.data_key = out.token;
     delete out.token;
   }
+
+  if (type === 'kount_inquiry') {
+  // default values for email and ANID when they weren't specified in payload
```
Comment (wuon) [inline index.js]: indent comment?

File: test/unit.test.js (hunk 1 above)
Comment (gabrielsimpsons) [reply test/unit.test.js:332]: Those are test keys taken from moneris

**Pull request merged**